contracts/vesting/src/vesting/account/mod.rs

```diff
@@ -231,18 +200,15 @@ impl Account {
         mix: IdentityKey,
         storage: &dyn Storage,
     ) -> Result<Uint128, ContractError> {
-        Ok(self
-            .delegations_for_mix(mix, storage)?
+        Ok(load_delegations_for_mix(self.storage_key(), &mix, storage)?
```

Comment:
Same as before. Another unbonded call : (
This thing is the reason there exists a memoised value for total delegation of particular mixnode. Perhaps it would also be appropriate here?

Reply:
Does it make a difference that we're not materialising all the values, but iterating and folding? Vesting contract does to have access to that mix node, what we're getting here are total_delegations of this particular vesting contract for a given mix node

Reply:
This is annoying, I need a way to just generate the pagination code...